packages/grafana-ui/src/components/BrowserLabel/Label.tsx

```diff
@@ -60,12 +77,16 @@ export const Label = forwardRef<HTMLElement, Props>(
         )}
         {...rest}
       >
-        <Highlighter
-          textToHighlight={text}
-          searchWords={searchWords}
-          autoEscape
-          highlightClassName={styles.matchHighLight}
-        />
+        {typeof highlightParts !== 'undefined' ? (
```

Comment:
Is this different than just saying `highlightParts !== undefined`?

Reply:
They're equivalent. I thought I was the typeof check somewhere else when working on this, but I think I just read it incorrectly. I'll update the check for consistency.